imageproxy.go

```diff
@@ -44,6 +44,10 @@ type Proxy struct {
 	// proxied from.  An empty list means all hosts are allowed.
 	Whitelist []string
 
+	// Blacklist specifies a list of domains that should not be possible
+	// to proxy from. An empty list means all domains are allowed.
```

Comment:
Second sentence should probably be changed to, "An empty list means no domains are blacklisted.", since it might not be the case that all are allowed, depending on whether there is a whitelist.  And a similar change to the Whitelist field above.